dashhive/dash-daypass
# HTML element based Solution

https://daypass.duckdns.org/

To test:

Pull and open index.html.

Restore original content with console command:
```
DashDayPass.removePaywall()
```

Also see updated Readme.md

|![Desktop - Light](https://user-images.githubusercontent.com/72463218/170392947-3daf9e2d-8e4f-48e3-aa37-878097f672e3.png)|![Desktop - Dark](https://user-images.githubusercontent.com/72463218/170393007-6986a738-0e1b-422f-b419-f4dd4baddee5.png)|![Mobile - Dark](https://user-images.githubusercontent.com/72463218/170393349-c2a07017-8d86-4afb-b67b-0d2f981275ee.png)|![Mobile - Light](https://user-images.githubusercontent.com/72463218/170393201-f803ba79-b302-4273-b768-7fd2747179a2.png)|
|---|---|---|---|
